cmake/Codegen.cmake

```diff
@@ -95,6 +95,26 @@ if(INTERN_BUILD_ATEN_OPS)
     endif()
     list(JOIN ROWWISE_SCALED_MM_FILE_COMPILE_FLAGS " " ROWWISE_SCALED_MM_FILE_COMPILE_FLAGS)
     set_source_files_properties(${ROWWISE_SCALED_MM_FILE} PROPERTIES COMPILE_FLAGS "${ROWWISE_SCALED_MM_FILE_COMPILE_FLAGS}")
+
+    set(ROWWISE_SCALED_MM_FILE "${CMAKE_CURRENT_LIST_DIR}/../aten/src/ATen/native/cuda/ScaledGroupMM.cu")
```

Comment:
Unrelated: It seems that these non portable arches are becoming and more important we might want to figure out a more generalizable approach